packages/slider/package.json

```diff
@@ -83,7 +83,7 @@
         "@spectrum-web-components/theme": "^0.40.3"
     },
     "devDependencies": {
-        "@spectrum-css/slider": "^4.1.19"
+        "@spectrum-css/slider": "4.2.1-beta.0"
```

Comment:
Have this gotten a full release?

Reply:
Not yet, on our side [this work](https://github.com/adobe/spectrum-css/pull/2325) is pending re-review now that we've validated the changes here. I'll reach out to the team again to try to get some eyes on it.